Client/src/components/contanier/staff/Courses/AddCourse/index.css

```diff
@@ -0,0 +1,169 @@
+.add-course {
+  width: 80%;
+  height: 641px;
+  display: flex;
+}
+.add-course-titel {
+  color: #06468f;
+  letter-spacing: 1px;
+  font-size: 34px;
+  font-weight: bold;
+  margin-top: 50px;
+  margin-left: 39px;
+  margin-bottom: 9px;
+}
+.add-course-line {
+  width: 204px;
+  border-bottom: 1.5px solid #06468f;
+  margin-top: 100px;
+  margin-left: 39px;
+  margin-bottom: 73px;
+}
+
+.left {
+  width: 45%;
+}
+.right {
+  width: 55%;
+}
+
+.course-name-label {
+  color: #06468f;
+  font-size: 14px;
+  margin-left: 82px;
+  margin-bottom: 7px;
+}
+.course-name-input {
+  width: 350px;
+  height: 39px;
+  background-color: #e0e0e0;
+  margin-left: 82px;
+  margin-bottom: 7px;
+  margin-top: 7px;
+  border: none;
+}
+
+.numbers-of-students-label {
+  color: #06468f;
+  font-size: 14px;
+  margin-left: 82px;
+  margin-bottom: 7px;
+}
+.numbers-of-students-input {
+  width: 350px;
+  height: 39px;
+  background-color: #e0e0e0;
+  margin-left: 82px;
+  margin-bottom: 7px;
+  margin-top: 7px;
+  border: none;
+}
+
+.description-label {
+  color: #06468f;
+  font-size: 14px;
+  margin-left: 82px;
+  margin-bottom: 7px;
+}
+.description-input {
+  background-color: #e0e0e0;
+  margin-left: 82px;
+  margin-bottom: 7px;
+  margin-top: 7px;
+  width: 350px;
+  height: 143px;
+  border: none;
+}
+.level-label {
+  flex-direction: column;
+  display: flex;
+  color: #06468f;
+  font-size: 14px;
+  margin-bottom: 7px;
+}
+.level-input {
+  width: 350px;
+  height: 39px;
+  background-color: #e0e0e0;
+  margin-bottom: 34px;
+  margin-top: 7px;
+  border: none;
+}
+
+.days-label {
+  display: flex;
+  flex-direction: column;
+  color: #06468f;
+  font-size: 14px;
+  margin-bottom: 7px;
+}
+.days-input {
+  width: 350px;
+  height: 39px;
+  background-color: #e0e0e0;
+  margin-bottom: 34px;
+  margin-top: 7px;
+  border: none;
+}
+.dates {
+  display: flex;
+}
+.start-label {
+  display: flex;
+  flex-direction: column;
+  color: #06468f;
+  font-size: 14px;
+  margin-bottom: 7px;
+  margin-right: 16px;
+}
+.start-input {
```

Comment:
`.start-input` and `.end-input`
`..start-label ` and `.end-label `
the same style for both !!
use one class for it 
don't repeat your code :+1: